metadata-models/src/main/pegasus/com/linkedin/dataprocess/DataProcessInstanceRunEvent.pdl

```diff
@@ -15,6 +15,10 @@ import com.linkedin.common.Urn
 record DataProcessInstanceRunEvent includes TimeseriesAspectBase, ExternalReference {
 
   @TimeseriesField = {}
+  @Searchable = {
+    "fieldType": "TEXT",
```

Comment:
This should either be removed or set to KEYWORD. The reason is that the value is of type enum which would default to KEYWORD.